tests/slow/src/test/scala/tests/sbt/SbtStepDapSuite.scala

```diff
@@ -4,9 +4,18 @@ import tests.SbtBuildLayout
 import tests.SbtServerInitializer
 import tests.debug.BaseStepDapSuite
 
-class SbtStepDapSuite
-    extends BaseStepDapSuite(
-      "sbt-debug-step",
+abstract class SbtStepDapSuite(
+    useVirtualDocuments: Boolean,
+    suiteNameSuffix: String
+) extends BaseStepDapSuite(
+      useVirtualDocuments,
+      s"sbt-debug-step-$suiteNameSuffix",
       SbtServerInitializer,
       SbtBuildLayout
     )
+
+class SbtStepDapSuiteSaveToDiskSuite
```

Comment:
I would avoid running the tests double and just run a single test case with changed useVirtualDocuments option. These tests do take a while and I think is is also partly the reason why Windows tests run out of memory at some point.